packages/hash/external-services/docker-compose.yml

```diff
@@ -120,7 +120,7 @@ services:
       HASH_GRAPH_PG_PORT: "5432"
       HASH_GRAPH_PG_DATABASE: "${HASH_GRAPH_PG_DEV_DATABASE}"
       HASH_GRAPH_LOG_FORMAT: "${HASH_GRAPH_LOG_FORMAT:-pretty}"
-      HASH_GRAPH_OTLP_ENDPOINT: "http://telemetry-collector:4317"
+      #      HASH_GRAPH_OTLP_ENDPOINT: "http://telemetry-collector:4317"
```

Comment:
If we want to disable tracing in this PR, should we add an explainatory comment as in https://github.com/hashintel/hash/blob/fb9a93c0a37e61cbc4580247cf53e4f883c10488/packages/hash/external-services/docker-compose.yml#L123-L125
and maybe even link to the task we've created?

Reply:
Good callout, I've updated the comment, cheers